docs/superpowers/plans/2026-04-12-mise-python-precompiled-arch.md

```
@@ -0,0 +1,383 @@
+@test "Unknown OS exits 1 with no output"
+{
+  _stub_uname "FreeBSD" "x86_64"
+  run sh "$SCRIPT"
+  [ "$status" -eq 1 ]
+  [ "$output" = "" ]
+}
+```
+
+- [ ] **Step 2: Run the tests to confirm they all fail (script not created yet)**
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)